### DISCUSSION TITLE
We found the patch versions not appeared in Golang Index.

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [dgraph-io/dgraph#8911] [Ideas]
Hello, we are a research team working on Golang. During our investigation, we discovered that the CVE-2023-31135 was addressed and the patch versions were released. However, we noticed that these patch versions have not appeared in the Golang Index, which means that 'go list' cannot automatically push the patch versions to downstream users.

We recommend that after releasing the versions, you push them to the Golang Index using the command 'go get github.com/dgraph-io/dgraph@version'. This will enable the automatic distribution of the patch versions to downstream users. Thank you for your attention.

**Comment by matthewmcneely** [comment]
Will be addressed in #9527.
